app/root.tsx

```diff
@@ -5,15 +5,21 @@ import { TopNav } from '~/components/layout/top-nav';
 import { Links, LiveReload, Meta, Outlet, Scripts, ScrollRestoration } from '@remix-run/react';
 import Footer from './components/layout/footer';
 import { getCurrentUser } from '~/modules/session/session.server';
+import { db } from './modules/database/db.server';
 
 export const links: LinksFunction = () => [{ rel: 'stylesheet', href: stylesheet }];
 
 export async function loader({ request }: LoaderArgs) {
+  const events = await db.event.findMany({ include: { group: true }, take: 24 });
```

Comment:
If these are recommendations, we might want to filter out all events and groups of the current user. So one user doesn't see their own groups and events as recommendations?

Reply:
Absolutely. We will consider it maybe in the next PR.